internal/diagnostics/descriptions/parsers/cssParser.ts

```diff
@@ -14,7 +14,14 @@ export const cssParser = createDiagnosticsCategory({
 	EXPECTED_IDENTIFIER: {message: markup`Expected an identifier`},
 	EXPECTED_SELECTOR: {message: markup`Expected a selector`},
 	EXPECTED_LBRACKET: {
-		message: markup`Expected a left bracket </emphasis>{</emphasis>`,
+		message: markup`Expected a left bracket <emphasis>{</emphasis>`,
```

Comment:
```suggestion
		message: markup`Expected a left curly bracket <emphasis>{</emphasis>.`,
```

We could show the character that we actually get instead of the bracket